modules/quant_purger/quant_purger.install

```diff
@@ -34,3 +34,23 @@ function quant_purger_schema() {
   ];
   return $schema;
 }
+
+/**
+ * Update configuration for module settings.
+ */
+function quant_purger_update_0001(&$sandbox) {
+  $config = \Drupal::configFactory()->getEditable('quant_purger.settings');
+  $config->set('tag_blocklist', [
+    'rendered',
+    'http_response',
+    '4xx-response',
+    'block_view',
+    'block_view_content',
+    'config:*',
+    'node_view',
+    'node_list',
+    'media_view',
```

Comment:
This doesn't completely match the settings file but maybe I'm unclear what's going on here.